src/gradient/typedefs.ts

```diff
@@ -0,0 +1,104 @@
+import { Percent } from "../typedefs";
+
+export type GradientUnits = 'pixels' | 'percentage';
+
+export type GradientType = 'linear' | 'radial';
+
+export type GradientValue = number | Percent | string;
```

Comment:
is this a value for the gradient coordinate?

Reply:
done